django/forms/widgets.py

```diff
@@ -268,6 +268,7 @@ class Input(Widget):
 
     def __init__(self, attrs=None):
         if attrs is not None:
+            attrs = attrs.copy()
             self.input_type = attrs.pop('type', self.input_type)
```

Comment:
Why the overhead of the copy? Surely changing `.pop()` to `.get()` is enough if `type` should persist?